OFEP-metric-hooks.md

```diff
@@ -0,0 +1,67 @@
+## OFEP-hook-for-metrics
+
+## State: PENDING REVIEW
+
+This OFEP propose to introduce OpenFeature hook for OpenTelemetry metrics.
+
+## Background
+
+We already have OpenTelemetry Span support through hooks. Similarly, we can provide a dedicated hook for 
+OpenTelemetry 
+metrics. Providing telemetry data out of the box will make OpenFeature attractive for both users and vendors. 
+
+## Proposal
+
+The proposal here is to define a set of metrics that can be used with different hook stages. For example, error metrics 
+can be collected at the hook's `error` stage. Going with this background, I propose the following metrics.
+
+### evaluationRequests
```

Comment:
Could you derive this metric by aggregating successes and failures?

Reply:
Yes, that's correct. evaluationRequests = evaluationSuccess + evaluationError

Note - I have updated the naming of metrics with latest commits